stable/prometheus/templates/node-exporter-podsecuritypolicy.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
     component: "{{ .Values.nodeExporter.name }}"
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: 'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName: 'runtime/default'
```

Comment:
shall we make this optional?

Reply:
I think it is a good practice to keep it by default but making this optional might be safer in case apparmor is not enabled.